org-formation/organization.yaml

```diff
@@ -385,6 +391,7 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         Department: IBC
         Project: Infrastructure
+        CostCenter: NO PROGRAM / 000000
```

Comment:
Since nextflow-tower supports multiple projects, see the granular budget report comment.  Do we leave out the CostCenter tag on the account level if there is already more granular budget reporting?